Makefile

```diff
@@ -31,10 +31,11 @@ precommit_install:  ## Install the project's precommit hooks from .pre-commit-co
 	@echo "Make sure to first run install_test first"
 	pre-commit install
 
-## Python Checkstyle
-.PHONY: lint
-lint:  ## Run pylint on the Python sources to analyze the codebase
-	find $(PY_SOURCE) -path $(PY_SOURCE)/metadata/generated -prune -false -o -type f -name "*.py" | xargs pylint
+isort:
+	isort $(PY_SOURCE) --skip $(PY_SOURCE)/metadata/generated --profile black --multi-line 3
+
+lint: ## Python Checkstyle
+	find $(PY_SOURCE) -path $(PY_SOURCE)/metadata/generated -prune -false -o -type f -name "*.py" | xargs pylint --ignore-paths=$(PY_SOURCE)/metadata_server/ --max-line-length=88 --extension-pkg-whitelist='pydantic'
```

Comment:
Hi @TeddyCr, let's remove `isort` from the changes as we have a joint recipe `py_format` that handles both calls 🙏